conduit/transforms/audio.py

```diff
@@ -0,0 +1,352 @@
+from typing import Callable, Optional, Union
+
+import numpy as np
+from ranzen import parsable
+import torch
+from torch import nn
+import torchaudio.transforms
+
+Mels = torchaudio.transforms.MelSpectrogram
+
+__all__ = ["LogMelSpectrogram", "LogMelSpectrogramNp", "Framing", "FramingNp"]
+
+
+class LogMelSpectrogram(torchaudio.transforms.MelSpectrogram):
+    @parsable
+    def __init__(
+        self,
+        sample_rate: int = 16000,
+        n_fft: int = 400,
+        win_length: Optional[int] = None,
+        hop_length: Optional[int] = None,
+        f_min: float = 0.0,
+        f_max: Optional[float] = None,
+        pad: int = 0,
+        n_mels: int = 128,
+        window_fn: Optional[Callable] = None,
+        power: float = 2.0,
+        normalized: bool = False,
+        wkwargs: Optional[dict] = None,
+        center: bool = True,
+        pad_mode: str = "reflect",
+        onesided: bool = True,
+        norm: Optional[str] = None,
+        mel_scale: str = "htk",
+        log_offset: float = 0.0,
+    ):
+        if window_fn is None:
+            window_fn = torch.hann_window
+
+        super().__init__(
+            sample_rate=sample_rate,
+            n_fft=n_fft,
+            win_length=win_length,
+            hop_length=hop_length,
+            f_min=f_min,
+            f_max=f_max,
+            pad=pad,
+            n_mels=n_mels,
+            window_fn=window_fn,
+            power=power,
+            normalized=normalized,
+            wkwargs=wkwargs,
+            center=center,
+            pad_mode=pad_mode,
+            onesided=onesided,
+            norm=norm,
+            mel_scale=mel_scale,
+        )
+        self.log_offset = log_offset
+
+    def forward(self, waveform: torch.Tensor) -> torch.Tensor:
```

Comment:
```suggestion
    def forward(self, waveform: torch.Tensor) -> torch.Tensor:
```
Can Tensor from torch (rather than accessed through it) for consistency?